pipeline/vae_ecg/main.py

```diff
@@ -85,5 +89,49 @@ def train(cfg: DictConfig) -> nn.Module:
     return model
 
 
+def forward_step(
+    model: nn.Module,
+    data: torch.Tensor,
+    beta: float,
+) -> dict[str, torch.Tensor]:
+    recon_batch, mu, logvar = model(data)
+
+    magic_factor = 0.1
```

Comment:
would it be a good idea to move the `magic_factor` to config?

Reply:
For sure, I only need to remember what it is first 🥲

Reply:
We had some trouble with exploding gradients in this model iirc, this might be the value to control it? Some implementations also used weights near the loss, it might be one of them?

Reply:
From what I recall it's about scaling the loss components, we had a discussion about it here: https://github.com/Nospoko/autoencoders-demo/pull/7 I wanted to get rid of the sum reduction, to get operate on per-sample values. I think chat gpt advised me to add a scaling factor of reconstruction loss :)